src/test/java/seedu/rc4hdb/model/venues/booking/RecurrentBookingTest.java

```diff
@@ -17,7 +17,7 @@ public class RecurrentBookingTest {
 
```

Comment:
We might have to make `Booking` be more defensive. I saw that you did not test for any null values. But that's because our `Booking`, does not have any null checks.

Reply:
Even though it is unlikely that we would ever have the case that it would clash with a `null` value.

Reply:
I have added the null checks to `Booking`. Do note that Naren is currently working on the rest of the test cases for `RecurrentBooking`. 

The existing test cases for `RecurrentBooking` were written during the groundwork for the venue feature, when `Booking` only had two public methods. See #110.